crates/hir_ty/src/tests/simple.rs

```diff
@@ -2564,3 +2564,46 @@ fn f() {
     "#,
     )
 }
+
+#[test]
+fn qualified_as_trait_cast() {
```

Comment:
Does this test belong here or in regressions? or maybe somewhere else?

Reply:
I guess it doesn’t matter if you add a cov-mark here.

Reply:
Do cov-marks not work across crates? The test(check) is in `hir_ty` but the fix(hit) is in `hir_def`

Reply:
Oups, indeed, cov-marks work only within a single crate. That's because they rely on `cfg(test)` to emit the code, and that is set only for the leaf crate.

Reply:
Ah that makes sense, figured it had to be something like that.